db_functions/partitioned_manage_trigger.sql

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
Please update new files with the Apache text:  Example: https://github.com/project-koku/koku/blob/ed866cb9c0f45e7aeb5ee0caab8e41fb8fe1ebdb/db_functions/clone_schema.sql#L3